src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -2131,6 +2134,65 @@ private void checkInternetPermission() {
         }
     }
 
+    protected String serialize(String uniqueUserId) throws UsageAuthenticationException {
+        if (StringExtensions.IsNullOrBlank(uniqueUserId)) {
+            throw new IllegalArgumentException("uniqueUserId");
+        }
+        
+        if ((new BrokerProxy(mContext)).canSwitchToBroker()) {
+            throw new UsageAuthenticationException(ADALError.FAIL_TO_EXPORT,"Failed to export the FID because broker is enabled.");
+        } else {
+            /* the current serialize/deserialize feature is for first party 
+             * so the client ID for the FoCI token cache item is hard coded below
+             */
+            final String foci = "1";
+            TokenCacheItem tokenItem = new TokenCacheItem();
+            final String cacheKey = CacheKey.createCacheKey(this.getAuthority(), null, foci, true, uniqueUserId);
+            tokenItem = this.getCache().getItem(cacheKey);
+            
+            if (tokenItem == null) {
+                Logger.i(TAG, "Cannot find the FoCI token cache item for this userID", "");
```

Comment:
Cannot be addressed. Because there is no function Logger.i(String tag, String message) found.